### DISCUSSION TITLE
Help! Rotate object (not auto-rotate)

### DISCUSSION THREAD

**Discussion by pragavame** [google/model-viewer#3471] [Q&A]
How to rotate 4 objects by 90 degrees?
My 4 objects have different IDs.
The code I use for one object doesn't work for others. Also, this code violates object plane coordinates. My object is in the zero plane, but this code makes the object come out of it and float in the air.
What's the right way to just make an object rotate 90 degrees?

```
<model-viewer class="model3d" id="model-1" style="width: 100%; height: 125%;" camera-orbit="90deg 90deg 90deg" camera-target="0m 0m 0m"  interpolation-decay="200" disable-zoom alt="A 3D model" shadow-intensity="1" shadow-softness="0" ar ar-modes="webxr scene-viewer quick-look" src="models/1.glb">
</model-viewer>

<model-viewer class="model3d" id="model-2" style="width: 100%; height: 125%;" camera-orbit="90deg 90deg 90deg" camera-target="0m 0m 0m" interpolation-decay="200" disable-zoom alt="A 3D model" shadow-intensity="1" shadow-softness="0" ar ar-modes="webxr scene-viewer quick-look" src="models/2.glb">
</model-viewer>



<script>
(() => {
  const modelViewer = document.querySelector('#model-1');
  const orbitCycle = [
    '90deg 90deg',
    modelViewer.cameraOrbit
  ];

  setTimeout(RotateObj, 1000);

  	function RotateObj() {
     	const currentOrbitIndex = orbitCycle.indexOf(modelViewer.cameraOrbit);
    	modelViewer.cameraOrbit = orbitCycle[(currentOrbitIndex) % orbitCycle.length];
  	}
})();
</script>

```

![2022-05-16_09-20-06](https://user-images.githubusercontent.com/99244643/168531285-2d588b7c-1ffe-4e3d-a049-095280511ab1.png)
![2022-05-16_09-16-11](https://user-images.githubusercontent.com/99244643/168531293-3596f1fa-1e05-4374-8d5a-308829d06c73.png)
![2022-05-16_09-14-47](https://user-images.githubusercontent.com/99244643/168531294-44b2118c-0960-4f1f-ae78-989a254ee997.png)

**Comment by samaneh-kazemi** [comment]
Hi @pragavame, thanks for reaching out. Can you please share a glitch link instead of copying the code here? It helps us to understand what's happening better and help you out to debug this.
